### PR TITLE
Adds a V1 implementation for Facebook key lookups

### PR DESCRIPTION
**Pull request by diracdeltas**
This is the implementation specified in `docs/fb-integration-design.md` as "Version 1".

cc @adon-at-work @sweis for review, thanks.

### CODE REVIEW

**Comment by diracdeltas** [comment]
Demo screencast: https://youtu.be/sMpo1t3amcI

**Comment by diracdeltas** [comment]
pinging @adon-at-work

**Comment by adon-at-work** [comment]
will give it a review within two days

**Comment by adon-at-work** [comment]
In general, it's a good to have feature. I'm done with the reviews. 
p.s. allowing comma-separated usernames can probably make it much more usable, or just wait for V2

File: docs/fb-integration-design.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+* In the case where Alice's key updates do not have to be signed by her
+previous key, a malicious provider can replace her
+key at any time. Alice can potentially complain about this if her client is
+online and checking key updates, but a determined attacker can keep her
+offline.
+
+### Facebook lookup guarantees:
+
+* Prevents equivocation by the key provider if Bob knows Alice's Facebook
+account username a priori and Alice has chosen to pubish her key to her
```
Comment (adon-at-work) [inline docs/fb-integration-design.md]: s/pubish/publish/

**Comment by adon-at-work** [comment]
:+1:

**Pull request merged**